_blog.yml

```
@@ -6239,3 +6239,14 @@
     - guide
     - community
     - open-source
+
+- local: async-inference
```

```suggestion
- local: async-robot-inference
```
make the URL more clearly about robotics (vs. general inference)

(or `robotics-async-inference`, etc)